src/core/interface.scala

```diff
@@ -1,9 +1,10 @@
 package magnolia1
 
-import language.experimental.macros
 import scala.annotation.tailrec
 import scala.reflect.*
 
+import language.experimental.macros
```

Comment:
Does this import do anything anymore in Scala 3?

Reply:
I don't think so, not sure how it end up here :D